README.md

```diff
@@ -117,7 +117,7 @@ The `Menubar` class is an event emitter:
 
 | menubar  | Electron                   | Notes                                                                                                                      |
 | -------- | -------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
-| 9.x.x    | 9.x.x \| 10.x.x \| 11.x.x \| 12.x.x \| 13.x.x \| 14.x.x \| 15.x.x                     |                                                                                                                            |
+| 9.x.x    | 9.x.x \| 10.x.x \| 11.x.x \| 12.x.x \| 13.x.x \| 14.x.x \| 15.x.x \| 15.x.x                     |                                                                                                                            |
```

Comment:
@MvRemmerden, this should probably be `16.x.x` :sweat_smile: 
```suggestion
| 9.x.x    | 9.x.x \| 10.x.x \| 11.x.x \| 12.x.x \| 13.x.x \| 14.x.x \| 15.x.x \| 16.x.x                     |                                                                                                                            |
```

Reply:
🤦‍♂️ You are obviously right, it's too late... 😁